### PR TITLE
initial StreamingHTTPChannel2023

### PR DESCRIPTION
**Pull request by elf-pavlik**

### CODE REVIEW

File: streaming-http-channel-2023.bs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+DPoP: <proof>
+Content-Type: application/ld+json
+```
+```jsonld
+{
+  "@context": ["https://www.w3.org/ns/solid/notification/v1"],
+  "type": "StreamingHTTPChannel2023",
+  "topic": "https://storage.example/resource",
+  "state": "opaque-state",
+  "expiration": "2023-12-23T12:37:15Z",
```
Comment (csarven) [inline streaming-http-channel-2023.bs:133]:
```suggestion
  "endAt": "2023-12-23T12:37:15Z",
```

File: streaming-http-channel-2023.bs [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+  "receiveFrom": "https://fetch.example/aca3cb0a-fb0a-4091-b9f8-8117d2cdb392"
+}
+```
+> Response to the POST request, including channel type and the receiveFrom URL.
+
+In JavaScript, a client can use the data in the response to establish a connection to the Fetch API.
+And define how to handle notifications
+
+```js
+const response = await fetch('https://fetch.example/aca3cb0a-fb0a-4091-b9f8-8117d2cdb392')
```
Comment (csarven) [inline streaming-http-channel-2023.bs:157]: In WebSocket, there is url, protocol parameters where the protocol is the IRI of the channel type. Do we need something along those lines for fetch? Or is the specific receiveFrom URL is intended to be unique for the channel, i.e., channel type need not be part of the request.

File: streaming-http-channel-2023.bs [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+            "Kjetil Kjernsmo"
+        ],
+        "href": "https://solidproject.org/TR/protocol",
+        "title": "Solid Protocol",
+        "publisher": "W3C Solid Community Group"
+    },
+    "Solid.Notifications": {
+        "authors": [
+            "Aaron Coburn",
+            "Sarven Capadisli"
```
Comment (csarven) [inline streaming-http-channel-2023.bs:196]:
```suggestion
        "editors": [
            "Sarven Capadisli"
        ],                
        "authors": [
            "Aaron Coburn",
            "Sarven Capadisli",
            "elf Pavlik",
            "Rahul Gupta"
```

File: streaming-http-channel-2023.bs (hunk 2 above)
Comment (elf-pavlik) [reply streaming-http-channel-2023.bs:157]: I assume `receiveFrom` to be unique for the channel. Probably this should be made explicit in this spec.

**Pull request merged**